code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -0,0 +1,458 @@
+	catchphrase = "R'BRTH"
+
+/obj/item/melee/touch_attack/ash_leech/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	playsound(user, 'sound/magic/demon_attack1.ogg', 75, TRUE)
+	if(ishuman(target))
+		var/mob/living/carbon/human/tar = target
+		if(tar.anti_magic_check())
+			tar.visible_message("<span class='danger'>Spell bounces off of [target]!</span>","<span class='danger'>The spell bounces off of you!</span>")
+			return
+	var/mob/living/carbon/C2 = user
+	if(isliving(target))
+		var/mob/living/L = target
+		L.adjustBruteLoss(20)
+		C2.adjustBruteLoss(-20)
+	if(iscarbon(target))
+		var/mob/living/carbon/C1 = target
+		C1.blood_volume -= 20
+		if(C2.blood_volume < BLOOD_VOLUME_MAXIMUM) //we dont want to explode after all
+			C2.blood_volume += 20
+		return
+
+/obj/effect/proc_holder/spell/targeted/projectile/dumbfire/rust_wave
+	name = "Patron's Reach"
+	desc = "Channels energy into your gauntlet - firing it results in a wave of rust being created in it's wake."
+	proj_type = /obj/projectile/magic/spell/rust_wave
+	charge_max = 350
+	clothes_req = FALSE
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "rust_wave"
+	action_background_icon_state = "bg_ecult"
+	invocation = "SPR'D TH' WO'D"
+	invocation_type = "whisper"
+
+/obj/projectile/magic/spell/rust_wave
+	name = "Patron's Reach"
+	icon_state = "eldritch_projectile"
+	alpha = 180
+	damage = 30
+	damage_type = TOX
+	hitsound = 'sound/weapons/punch3.ogg'
+	trigger_range = 0
+	ignored_factions = list("e_cult")
+	range = 15
+	speed = 1
+
+/obj/projectile/magic/spell/rust_wave/Moved(atom/OldLoc, Dir)
+	. = ..()
+	playsound(src, 'sound/items/welder.ogg', 75, TRUE)
+	var/list/turflist = list()
+	var/turf/T1
+	turflist += get_turf(src)
+	switch(Dir)
+		if(NORTH)
+			T1 = get_step(src,WEST)
+			turflist += T1
+			turflist += get_step(T1,WEST)
+			T1 = get_step(src,EAST)
+			turflist += T1
+			turflist += get_step(T1,EAST)
+		if(SOUTH)
+			T1 = get_step(src,WEST)
+			turflist += T1
+			turflist += get_step(T1,WEST)
+			T1 = get_step(src,EAST)
+			turflist += T1
+			turflist += get_step(T1,EAST)
+		if(WEST)
+			T1 = get_step(src,NORTH)
+			turflist += T1
+			turflist += get_step(T1,NORTH)
+			T1 = get_step(src,SOUTH)
+			turflist += T1
+			turflist += get_step(T1,SOUTH)
+		if(EAST)
+			T1 = get_step(src,NORTH)
+			turflist += T1
+			turflist += get_step(T1,NORTH)
+			T1 = get_step(src,SOUTH)
+			turflist += T1
+			turflist += get_step(T1,SOUTH)
+
+	for(var/X in turflist)
+		if(prob(25))
+			continue
+		var/turf/T = X
+		T.rust_heretic_act()
+
+/obj/effect/proc_holder/spell/targeted/projectile/dumbfire/rust_wave/short
+	name = "Small Patron's Reach"
+	proj_type = /obj/projectile/magic/spell/rust_wave/short
+
+/obj/projectile/magic/spell/rust_wave/short
+	range = 7
+	speed = 2
+
+/obj/effect/proc_holder/spell/pointed/ash_cleave
+	name = "Cleave"
+	desc = "Causes severe bleeding on a target and people around him"
```

Comment:
Nitpicky but use neutral pronouns. Instead of 'him', use 'them'.

Reply:
good catch

Reply:
i think i fixed all of em